### DISCUSSION TITLE
Financial Events Filtered by Marketplace

### DISCUSSION THREAD

**Discussion by bry221996** [amzn/selling-partner-api-models#2829] [General] [answered]
Is there a way to filter financialEvents by marketplace ? 

https://developer-docs.amazon.com/sp-api/docs/finances-api-reference

Need to identify which financial events happen in specific marketplace. thanks

**Comment by wanglei0214** [comment, marked as answer]
Hi @bry221996 

it is currently unavailable to filter financialEvents by marketplace.
We have aware of it and have actively communicated with the product team to request this feature.
Please keep an eye on the our doc site for new announcement if once it is featured in the future.
